app/tests/unit/test_cli.py

```diff
@@ -116,38 +122,18 @@ def test_initialize_invalid_args(cli_runner: CliRunner):
     act: when _parse_args is called.
     assert: Error output is printed.
     """
-    result = cli_runner.invoke(main, args=["init"])
+    result = cli_runner.invoke(main, args=[*REQUIRED_MAIN_INPUTS, "init"])
 
     assert "Error: Missing option '--arch'" in result.output
 
 
-@pytest.mark.parametrize(
-    "invalid_args",
-    [
-        pytest.param({"": ""}, id="empty cloud name positional argument"),
-        pytest.param({" ": ""}, id="empty image name positional argument"),
-    ],
-)
-def test_invalid_latest_build_id_args(
-    cli_runner: CliRunner, latest_build_id_inputs: dict, invalid_args: dict
-):
+def test_invalid_latest_build_id_args(cli_runner: CliRunner):
     """
-    arrange: given invalid latest-build-id action arguments.
-    act: when _parse_args is called.
+    arrange: none.
+    act: when latest-build-id is called with no image name.
```

Comment:
can you explain the when pattern in the sentence? I would phrase it "call latest-build-id with no image name"

Reply:
I will update it with the next PR (since the tests have passed already!) Thanks!